lib/experimental/Forms/AvatarNameSelector/avatar-name.factory.tsx

```diff
@@ -1,149 +1,151 @@
+import avatar from "../../../../storybook-assets/avatar.jpeg"
 import { AvatarNamedEntity, AvatarNamedSubEntity } from "./types"
 
 export const famousEmployees: AvatarNamedEntity[] = [
   {
     id: 1000,
     name: "Albert Einstein",
-    avatar: "https://i.pravatar.cc/301",
```

Comment:
I think the problem is the misleading on the url path `/301`, `/302` etc. This is the avatar size, not the id of the avatar
You could use https://i.pravatar.cc/150?u=[X] replacing [X] with any value (ex: https://i.pravatar.cc/150?u=1) and for the same X value the image will be the same

Check: https://pravatar.cc/